recipes/raider/meta.yaml

```
@@ -0,0 +1,83 @@
+{% set name = "RAiDER" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dbekaert/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 18fb20fd9c250b168459c5258c66d7392cd58c5243831b6f0ece9c90ea3a72e4
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [py<37 or win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - python
+    - pip
+    - pybind11 >=2.5.0
+    - setuptools
+    - numpy
+    - cython
+
+  run:
+    - python
+    - cython
```

```suggestion
    - cython
```
Are your sure that cython is need at runtime? Looks like it might only be used to compile extension modules.